### ISSUE TITLE
Bad contrast 

### ISSUE THREAD

**Issue by orowith2os** [cotes2020/jekyll-theme-chirpy#1095]
## Describe the bug

Chirpy does not have very good contrast when it comes to accessibility. 

## To Reproduce

Use the default Chirpy theme.

## Expected behavior

Content should be easily readable by people of all abilities.

## Logs/Screenshots

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/93224879/439d9830-e6f2-4978-be10-6d6caf50545b)

## Additional context

Comparing contrast using https://www.whocanuse.com, Chirpy fails when it comes to blockquotes, and can improve when it comes to normal text.

In https://github.com/cotes2020/jekyll-theme-chirpy/issues/447, it's stated that "The low color contrast ratio is to prevent the sub-title (tagline) from getting the first attention." It may be possible to increase contrast more while not taking attention away from other content, and Chirpy should aim to do this.

Related PR: https://github.com/cotes2020/jekyll-theme-chirpy/pull/448

**Comment by cotes2020** [comment]
If you would like to submit a PR to improve the contrast of the text to WCAG 2.0 level AA, that would be great.

**Comment by orowith2os** [comment]
Still relevant.
